echo/test/unit/echo_client_test.js

```diff
@@ -229,14 +229,115 @@ describe("Echo Node Client Test Suite", function(){
 
             echoClient.addEvents('secret', {class:'class', source:'source'}, function(err, result){
 
-                (err === null).should.be.true;
+                (err === null).should.be.true();
 
                 result.body.timestamp.should.equal(1324524509);
                 result.body.user.should.equal("1234-5678-9012-3456");
                 result.body.source.should.equal("rl-app");
                 result.body.props.should.be.an.object;
                 result.body.props.url.should.equal("https://foo.bar/baz.html");
 
+                done();
+            });
+        });
+        it("- rounds the timestamp to nearest whole number for arrays of events", function(done){
```

Comment:
Whats the behaviour if the `timestamp` is not set?

Reply:
it does nothing, I'll add a test, if we agree on the changes here